client/question/question.js

```diff
@@ -341,9 +341,9 @@ tie.constant('WRONG_LANGUAGE_ERRORS', {
       {
         type: 'text',
         content: [
-          "Note that, in Python, we write boolean values as True and False. ",
-          "Double-check to make sure 'True' or 'False' are written with the ",
-          "letters in the correct case."
+          "Python only allows True and False for boolean values. Double-check ",
+          "to make sure 'True' or 'False' are written with the letters in the ",
```

Comment:
"with the letters in the correct case" seems a bit stilted to me, but I can't think of anything better. If you can, go for it, otherwise, approved.

Reply:
The more streamlined alternatives I can think of suffer from a consequent reduction in clarity, so on balance I think I'll leave it as is.